File: Leetcode/1550.go
package Leetcode

func threeConsecutiveOdds(arr []int) bool {
	n := len(arr)
	for i := 0; i < n-2; i++ {
		if arr[i]%2 == 1 && arr[i+1]%2 == 1 && arr[i+2]%2 == 1 {
			return true
		}
	}
	return false
}

File: Leetcode/1550.py
from typing import List


class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        n = len(arr)
        for i in range(n - 3):
            if arr[i] & 1 and arr[i + 1] & 1 and arr[i + 2] & 1:
                return True
        return False
